tests/unit_tests/test_wave_additional.py

```diff
@@ -0,0 +1,46 @@
+import pytest
+from swap_meet.vendor import Vendor
+from swap_meet.item import Item
+
+def test_get_newest_item_returns_item():
```

Comment:
Fantastic additional test wave for the optional enhancement! Great coverage test suite!